azerothcore/azerothcore-wotlk
# fix(DB/gameobject): The Scarlet Key should be lootable by all

## Changes Proposed:
-  Lower the respawn rate of Doan's Strongbox from 24 hours to 2 seconds

## Issues Addressed:

Doan's Strongbox inside the Scarlet Monastery, for the Scarlet Key, can only be looted by one (unless you sit around for 24 hours and wait for it to respawn).

The chest should be lootable by all. Reference [wowwiki](https://wowwiki.fandom.com/wiki/The_Scarlet_Key).



## Tests Performed:
-
-



## How to Test the Changes:


- .go c id 6487
- At the end of the wall, opposite the entrance to the room, the chest can be found. Open it and take the key.
- Without the fix, wait for 24 hours and see it respawns.

- Apply the fix, notice how the chest spawns almost instantly.

## Known Issues and TODO List:



## Target Branch(es):








 
## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
